pkg/didcomm/common/service/service.go

```diff
@@ -20,16 +20,24 @@ type DIDComm interface {
 	Event
 }
 
-// DIDCommMsg did comm msg
+// DIDCommMsg is used by handlers/dispatchers/events to pass DIDComm messages.
 type DIDCommMsg struct {
+	// message type
+	Type string
+	// message
+	Payload []byte
+}
+
+// Msg is used for outbound related features.
```

Comment:
There is a boolean inside this struct that could mean incoming?

Reply:
The boolean is not set anywhere outside the service layer. It look to manipulate the state machine operations. e.r. [here](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/protocol/didexchange/states.go#L218).

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/0b5ffafa1afbb9931651ae126eeaf822901dda9f..5826f9a5bccaf6a6bfdbe2bfa90f56f34194147e#diff-75342fbf986feea5a6bb6a33d0caff76R31) the comment (added state machine).